docs/quickstart-guide.md

```diff
@@ -0,0 +1,102 @@
+
+# Casper Fungible Token Quick Start Guide
+
+This quick start guide introduces you to the Casper client commands and Wasm files necessary to deploy a CEP-18 Casper Fungible Token contract to a [Casper network](https://cspr.live).
+
+The [Ethereum Request for Comment (ERC20)](https://eips.ethereum.org/EIPS/eip-20#specification) standard defines a set of rules that dictate the total supply of tokens, how the tokens are transferred, how transactions are approved, and how token data is accessed. These fungible tokens are blockchain-based assets that have value and can be transferred or recorded.
+
+To execute transactions on a Casper network (involving fungible tokens), you will need some CSPR tokens to pay for the transactions.
+
+For greater detail into the creation and mechanics of the Casper fungible token contract, see the full [Casper Fungible Token Tutorial](https://github.com/casper-ecosystem/cep18/blob/master/TUTORIAL.md).
+
+## Prerequisites
+
+Before using this guide, ensure you meet the following requirements:
+
+-   Set up your machine as per the [prerequisites](https://docs.casper.network/developers/prerequisites/)
+-   Use the [Casper command-line client]
+-   Get a valid [`node-address`](https://cspr.live/tools/peers)
+-   Know how to deploy a [smart contract](https://docs.casper.network/developers/dapps/sending-deploys/) to a Casper network
+-   Hold enough CSPR tokens to pay for transactions
+
+# Setup
+
+Clone the [fungible token (CEP-18) contract repository](https://github.com/casper-ecosystem/cep18) and run the `make build-contract` command. This will create the `cep18.wasm` and the `cep18_test_contract.wasm`. The token Wasm is the main contract. We will use the `test_contract` contract Wasm to query the balances and allowances of the fungible token balances throughout this workflow.
```

Comment:
```suggestion
Clone the [fungible token (CEP-18) contract repository](https://github.com/casper-ecosystem/cep18) and run the `make build-contract` command. This will create the `cep18.wasm` and the `cep18_test_contract.wasm`. The token Wasm is the main contract. We will use the `cep18_test_contract` Wasm to query the balances and allowances of the fungible token balances throughout this workflow.
```